cbf/v1/cbf-v1.0.0.yaml

```diff
@@ -0,0 +1,556 @@
+  license:
+    url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
+    name: Apache 2.0
+  contact:
+    name: Digital Container Shipping Association (DCSA)
+    url: 'https://dcsa.org/'
+    email: info@dcsa.org
+servers:
+  - url: 'https://dcsa.org/cbf'
+tags:
+  - name: Vessel Operator
+    description: ' '
+  - name: VSA Partner
+    description: ' '
+paths:
+  /v1/booking-forecasts:
+    parameters: []
+    get:
+      summary: Fetch Booking Forecast
+      tags:
+        - VSA Partner
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BookingForecast'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/BookingForecast'
+          headers:
+            API-Version:
+              $ref: ../../models/Headers.yaml#/components/headers/API-Version
+        '400':
+          description: Invalid request for booking forecast.
+          content:
+            application/json:
+              schema:
+                $ref: ../../models/ErrorResponse.yaml
+              examples:
+                Invalid GET Request:
+                  value:
+                    httpMethod: GET
+                    requestUri: 'https://dcsa.org/dcsa/cbf/v1/booking-forcast'
+                    statusCode: 400
+                    statusCodeText: Bad Request
+                    errorMessage: Either carrierServiceCode or universalServiceReference must be provided.
+                    providerCorrelationID: 4426d965-0dd8-4005-8c63-dc68b01c4962
+                    errorDateTime: '2019-11-12T07:41:00+08:30'
+                    errors: []
+        '404':
+          description: Booking forecast not foound.
+          content:
+            application/json:
+              schema:
+                $ref: ../../models/ErrorResponse.yaml
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: ../../models/ErrorResponse.yaml
+      operationId: get-booking-forecast-by-reference
+      description: |-
+        Fetch the booking forecast by either the `carrierServiceCode` or the `universalServiceReference`.\
+        Either one is **REQUIRED** but not both when fetching the booking forcast.
+      parameters:
+        - schema: {}
+          in: query
+          name: carrierServiceCode
+          description: |-
+            The carrier-specific code of the service for which the schedule details are published.\
+            **REQUIRED** if `universalServiceReference` is not provided.
+        - schema:
+            type: string
+          in: query
+          name: universalServiceReference
```

Comment:
You are not imposing any requirements to this? (regEx)

Reply:
Nice catch, missed it in the param. Will add it.